emily/handler/src/api/handlers/deposit.rs

```diff
@@ -300,3 +339,30 @@ pub async fn update_deposits(
 }
 
 // TODO(393): Add handler unit tests.
+
+// Test module
+#[cfg(test)]
+mod tests {
+
+    use super::*;
+    use sbtc::testing::{self, deposits::TxSetup};
+    use test_case::test_case;
+
+    #[test_case(15000, 500_000, 150; "All parameters are normal numbers")]
+    #[test_case(0, 0, 0; "All parameters are zeros")]
+    fn test_scripts_to_resource_parameters(max_fee: u64, amount_sats: u64, lock_time: u32) {
```

Comment:
nit: why `lock_time: u32`, given that below is used as i64 and u64?

Reply:
Because `tx_setup` still takes an `i64`, and only an unsigned 32 or fewer bit number can always be cast to both a `u64` and a `i64`